src/Akka.Persistence.Sql/Config/ReadJournalConfig.cs

```diff
@@ -31,6 +32,8 @@ public ReadJournalConfig(Configuration.Config config)
             WriteIsolationLevel = IsolationLevel.Unspecified;
         }
 
+        public string WritePluginId { get; }
```

Comment:
Adding the missing setting as a property in `ReadJournalConfig` as its supposed to